cmake/ctest/drivers/atdm/vortex

```diff
@@ -0,0 +1 @@
+ats2
```

Comment:
What is this?

Reply:
This is from https://github.com/trilinos/Trilinos/pull/6547/commits/25c32e04a5a25238fc13a6e5d96eeb0eb3818781. It's just a symlink: vortex->ats2

Reply:
When setting `hostnameMatchSystemName=ats2` you can delete this symlink.